src/aiy/i18n.py

```diff
@@ -17,13 +17,13 @@
 import gettext
 import os
 
-DEFAULT_LANGUAGE_CODE = 'en-US'
+_DEFAULT_LANGUAGE_CODE = 'en-US'
 
-LOCALE_DIR = os.path.realpath(
-    os.path.join(os.path.abspath(os.path.dirname(__file__)), '../po'))
-LOCALE_DOMAIN = 'voice-recognizer'
+_LOCALE_DIR = os.path.realpath(
+    os.path.join(os.path.abspath(os.path.dirname(__file__)), '../../po'))
```

Comment:
Please add a TODO, since this becomes a library, we cannot hardcode the path to the po dir. We'd need a constructor (or some init method).